src/main/java/ch/njol/skript/expressions/ExprBreedingFamily.java

```diff
@@ -0,0 +1,97 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityBreedEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Breeding Family")
+@Description("Represents a family within a breeding event")
+@Examples({
+	"on breeding:",
+	"\tsend \"When a %mother% and %father% love each other they make a %offspring%\" to breeder"
+})
+@Since("INSERT VERSION")
+public class ExprBreedingFamily extends SimpleExpression<LivingEntity> {
+
+	static {
+		Skript.registerExpression(ExprBreedingFamily.class, LivingEntity.class, ExpressionType.SIMPLE,
+			"[breed[ing]] mother",
+			"[breed[ing]] father",
+			"[breed[ing]] (offspring|child)",
```

Comment:
I think `[breed[ing]]` should be compulsory, `%mother% %father%` etc. sounds really weird without context

Reply:
Maybe, I'll go head and change these in a bit